rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+
+  def initialize(person, book, date)
+    @date = date
+    @person = person
+    @book = book
+
+    person.rentals << self
+    book.rentals << self
```

Comment:
Great job implementing `belongs_to` that's why I assumed in `book.rb` that we forgot to add `has_many` relationship ☺️

Reply:
Thank you very much for your feedback @Amrhub. Based on the last comment, does that mean [this change](https://github.com/AdornChoga/oop-school-library/pull/3#discussion_r843127254) is the only required change?

Reply:
You are most welcome @AdornChoga, any comment with the checkbox at the start is the changes we need to make actually this comment you linked is supposed to be OPTIONAL I changed it, so kindly also make association `has_many` between book to rental and between person to rental

Reply:
Thank you for the clarification @Amrhub. I'm doing that right away.